docs-gluu-server-prod/docs/mkdocs.yml

```diff
@@ -72,6 +72,7 @@ nav:
 - Administration Guide:
         - 'oxTrust admin UI': 'admin-guide/oxtrust-ui.md'
         - 'Interception Scripts': 'admin-guide/custom-script.md'
+        - 'Logs delete': 'admin-guide/logs-delete.md'
```

Comment:
There is dedicated page **Log Management** (`operation/logs.md`) which can be used instead of using separate page. We can add new section there.